### DISCUSSION TITLE
关于战眼(battleye)扫内存反外挂

### DISCUSSION THREAD

**Discussion by ZengXiaoPi** [LocalizeLimbusCompany/LocalizeLimbusCompany#65] [常规 | General]
## 信息
1. 目前最保险的办法是卸载MelonLoader及mod，办法为重命名**version.dll**或直接删除
2. 一旦被封禁不仅仅代表账号被封禁，被封禁后整个设备将无法游玩使用了白眼反作弊的游戏，需要更换硬盘才能解除封禁
3. ~~这货真的是眼线+爪牙。。。~~
4. **注意，根据MIT协议，零协会不对账号的封禁等等由于汉化补丁导致您的损失负责**
> 在任何情况下，都不得作者或版权所有者对任何索赔、损害赔偿或其他索赔、损害赔偿或其他责任责任，无论是在合同、侵权或其他诉讼中，由以下原因引起：出于或与本软件有关，或在软件。<br>摘自[MIT协议](https://github.com/LocalizeLimbusCompany/LocalizeLimbusCompany/blob/master_ilcpp/LICENSE)

### 使用了白眼的游戏（仅供参考）
```
Unturned 未转变者
PUBG 绝地求生
彩六
ARMA
H1Z1
方舟进化
……
```

欢迎补充以及提出解决方法。

**Comment by ZengXiaoPi** [comment]
~~非常有名的游戏，PUBG就用的这货。。。~~

**Reply by N071** [reply]
推特上EN是有在骂用了BE 甚至反作弊本身的
但是木毕竟已成舟 月计钱都给白眼交了

**Comment by ZengXiaoPi** [comment]
即使目前开发成本较高，本仓库也可以作为一个存储翻译文本的仓库。

**Comment by xgdjgo** [comment]
先卸了，希望能让用，方便
